### PR TITLE
Added Nexus example

### PR DESCRIPTION
**Pull request by tobilg**
As per https://github.com/mesosphere/universe/pull/813 I added a Nexus 3 example.

### CODE REVIEW

**Comment by mhausenblas** [comment]
OK, cool, thanks @tobilg … watching 813 and once that is merged, this one here is merged as well ;)

File: 1.8/nexus/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+[Nexus 3](http://www.sonatype.org/nexus/) is a repository manager that supports a broad variety of package managers, namely Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories. DC/OS allows you to quickly configure, install and manage NGINX.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Anyone interested in running a repository manager.
+- Scope: Learn how to install Nexus 3 on DC/OS
+
+## Prerequisites
+
+- A running DC/OS 1.8 cluster with at least 1 node having at least 0.5 CPUs and 2 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.8/usage/cli/install/) installed.
```
Comment (ryadav88) [inline 1.8/nexus/README.md]: Is `marathon-lb` a prereq. If yes, add that here

File: 1.8/nexus/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+The above `options.json` file configures Nexus 3 as follows:
+
+- `name`: This parameter configures the name of the service itself.
+- `cpus`: This parameter configures the number of CPU share to allocate to Nexus 3.
+- `mem`: This parameter configures the amount of RAM to allocate to Nexus 3.
+- `role`: The role which should be used to launch the service. Default is `*`.
+- `local-volumes`: This parameter configures whether service should use local host volumes. If so, the following properties need to be defined as well:
+ - `host-volume`: This is the folder/path which will be mounted in the container, and used to persist the Nexus 3 data on. You need to make sure that the folder exists on the host you used for host pinning, and that you did a 'chown -R 200:200' on the folder respectively.
```
Comment (ryadav88) [inline 1.8/nexus/README.md]: I would explain the steps of ssh'ing in to the host, and changing permissions.

**Comment by ryadav88** [comment]
@mhausenblas Universe PR is merged. this is ready for review.

**Comment by mhausenblas** [comment]
Awesome, thanks! Now, looks that in the mean time things have changed in the index. @tobilg can I please ask you to resolve that (small) issue and then I'm gonna merge it.

File: 1.8/nexus/README.md (hunk 1 above)
Comment (tobilg) [inline 1.8/nexus/README.md]: It isn't a prerequisite to use `marathon-lb`.

**Comment by tobilg** [comment]
Hmmm, any idea how I can see the actual conflicts? I merged my fork's master with your upstream/master. Still it seems to be marked as conflicting... Thanks!

**Comment by tobilg** [comment]
Sorry, too early this morning :-) Had to merge `upstream/master` to my `nexus` branch. Now everything look fine. Thanks!

**Pull request merged**